_data/wcag-act-rules/act-implementations.yml

```diff
@@ -159,3 +159,11 @@
   standards:
     - WCAG 2.0 Level A, AA
   jsonReport: https://raw.githubusercontent.com/Section508Coordinators/act-tt/main/act-trusted-tester.json
+
+- name: Webmate Accessibility Workbench
+  vendor: Testfabrik Consulting + Solutions AG
+  uniqueKey: webmate-accessibility-workbench
+  type: Automated tool
+  standards:
+    - WCAG 2.2 Level A, AA
+  jsonReport: https://webmedia.testfabrik.com/act_implementation_report/webmate_accessibility_workbench_2024_11_26.json
```

Comment:
@daniel-montalvo This looks like a dated report (otherwise, the filename is misleading). Isn't this a concern for updates? I notice that the other JSON report links are generic.

Reply:
Thanks for pointing this out. I'll go back to them on their issue and point it out there. We should indeed have a permanent for this.